test/metabase/api/database_test.clj

```diff
@@ -330,6 +337,22 @@
         (is (partial= {:cache_ttl 13}
                       (create-db-via-api! {:cache_ttl 13})))))))
 
+(deftest create-db-succesful-track-snowplow-test
+  (mt/test-drivers (disj (mt/normal-drivers) :h2)
+    (mt/with-model-cleanup [:model/Database]
+      (snowplow-test/with-fake-snowplow-collector
+        (let [dataset-def (tx/get-dataset-definition (data.impl/resolve-dataset-definition *ns* 'avian-singles))]
+          (mt/user-http-request :crowberto :post 200 "database"
+                                {:name    (mt/random-name)
+                                 :engine  driver/*driver*
+                                 :details (tx/dbdef->connection-details driver/*driver* nil dataset-def)}))
+        (is (=? {"database"     (name driver/*driver*)
+                 "database_id"  #hawk/malli :int
+                 "source"       "admin"
+                 "dbms_version" #hawk/malli [:fn some?]
+                 "event"        "database_connection_successful"}
```

Comment:
`=?` already supports functions as values which get called to do asserts, you don't need malli for this
```suggestion
        (is (=? {"database"     (name driver/*driver*)
                 "database_id"  int?
                 "source"       "admin"
                 "dbms_version" some?
                 "event"        "database_connection_successful"}
```

Reply:
Also you should use `string?` instead of `some?`